app/adapters/signal_adapter/inbound.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+module SignalAdapter
+  UNKNOWN_CONTRIBUTOR = :unknown_contributor
+  UNKNOWN_CONTENT = :unknown_content
+
+  class Inbound
+    UNKNOWN_CONTENT_KEYS = %w[mentions attachments contacts].freeze
```

Comment:
Where did you find the unknown content keys? We did not find any documentation, but according to this [implementation file](https://github.com/AsamK/signal-cli/blob/master/src/main/java/org/asamk/signal/json/JsonDataMessage.java) e.g. a sticker would have a dedicated sticker property.

Reply:
I just had a look into the JSON that gets parsed :laughing: and I used all the buttons that were available in my Signal app.